README.md

```diff
@@ -39,7 +38,7 @@ toolkit](https://software.intel.com/en-us/openvino-toolkit).
 
 OpenVINO™ Training Extensions provides a "model template" for every supported task type, which consolidates necessary information to build a model.
 Model templates are validated on various datasets and serve one-stop shop for obtaining the best models in general.
-If you are an experienced user, you can configure your own model based on [torchvision](https://pytorch.org/vision/latest/index.html), [pytorchcv](https://github.com/osmr/imgclsmob), [mmcv](https://github.com/open-mmlab/mmcv) and [OpenVINO Model Zoo (OMZ)](https://github.com/openvinotoolkit/open_model_zoo).
+If you are an experienced user, you can configure your own model based on [torchvision](https://pytorch.org/vision/stable/index.html), [pytorchcv](https://github.com/osmr/imgclsmob), [mmcv](https://github.com/open-mmlab/mmcv) and [OpenVINO Model Zoo (OMZ)](https://github.com/openvinotoolkit/open_model_zoo).
```

Comment:
Why don't we use the `latest` as doc links for develop branch?

Reply:
@goodsong81 `latest` link seems to be changed to `unstable` because there is only two different types of doc link for the `torchvision`, stable vs unstable